.github/workflows/cpp-python-tests.yml

```diff
@@ -7,102 +7,116 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-20.04
     strategy:
       matrix:
         python-version: [3.8]
+        build-type: [Debug, Release]
```

Comment:
Tests do not seem to work in Debug.

Reply:
Currently working on the getting the tests to work

Reply:
If we merge the other PR that fix the deadlocks, I think the tests will work in Debug.

Reply:
Is it the Mac build PR?

Reply:
Yes

Reply:
Looks like it can deadlock on release too.

Reply:
I agree. @mamaheux, we merge the mac branch now ?